less/styles.less

```diff
@@ -20,7 +20,12 @@
 .btn {
   text-transform: uppercase;
 }
-
+.toolbar-container-wrapper {
+    display:none;
+}
+#header-left {
```

Comment:
We have a class for this use case .header-padding. We should automatically add this class if the toolbar is disabled.